translate/storage/base.py

```diff
@@ -675,11 +675,8 @@ def insert_unit(source):
             insert_unit(unit.source)
 
         for location in unit.getlocations():
-            if location in self.locationindex:
-                # if sources aren't unique, don't use them
-                #FIXME: maybe better store a list of units like sourceindex
```

Comment:
There is a reason we set this to None.  The expected results are indeterminate.  But this now changes it to best effort but the commit forgets that this impacts ALL formats and communicates that its po2moz specific.

I'd actually comment with the reason.  Otherwise in a perfect world, you @unho, are going to be made to debug all future brokeness because you failed to caveat this commit.

Please don't remove useful info if you actually aren't fixing things.

I'm wondering if you thought though the option listed in the FIXME? That would possibly allow us to handle the dups in po2prop.  But could have oher bigger fallout.